prow/config/prow-config-documented.yaml

```diff
@@ -904,6 +904,31 @@ plank:
 pod_namespace: ' '
 
 
+# ProwJobDefaultEntries holds a list of defaults for specific values
+# Each entry in the slice specifies Repo and CLuster regexp filter fields to
+# match against the jobs and a corresponding ProwJobDefault . All entries that
+# match a job are used. Later matching entries override the fields of earlier
+# matching entires.
+prowjob_default_entries:
```

Comment:
I would make it more obvious that cluster and repo are filter clauses that then apply a tenant_id

Reply:
I want to keep the naming consistent with `default_decoration_config_entries` as the behavior is the same.

Reply:
Configuration (like `tenant_id`) is distinguished from filter fields (`cluster` and `repo`) by being nested under `config`.